tests/PHPStan/Rules/Methods/AbstractMethodInNonAbstractClassRuleTest.php

```diff
@@ -20,15 +20,22 @@ protected function getRule(): Rule
 
 	public function testRule(): void
 	{
-		$this->analyse([__DIR__ . '/data/abstract-method.php'], [
+		$this->analyse([
+			__DIR__ . '/data/abstract-method.php',
+			__DIR__ . '/data/bug-4244.php',
```

Comment:
Please add a separate test method for testing a new file.

Reply:
Done.